.github/scripts/verify_changes.sh

```diff
@@ -17,15 +25,32 @@ else
   base_commit=$(git log origin/$ref_name --oneline | head -2 | awk 'NR==2 {print $1}')
 fi
 
-# git diff with ... shows the differences count between base_commit and head_commit starting at the last common commit
-change_count=$(git diff $base_commit...$head_commit --name-only | awk -F"/" '{ print $1}' | uniq | wc -l)
+# git diff with ... shows the differences between base_commit and head_commit starting at the last common commit
+changed_dir=$(git diff $base_commit...$head_commit --name-only | awk -F"/" '{ print $1}' | uniq)
+change_count=$(echo "$changed_dir" | wc -l)
 
-if [[ $change_count -eq 1 ]]; then
-  changed_dir=$(git diff $base_commit...$head_commit --name-only | awk -F"/" '{ print $1}' | uniq)
-fi
+# There are 4 main conditions to check:
+#
+# 1. more than two changes found, set the flags to false
+# 2. doc only change
+# 3. ui only change
+# 4. two changes found, if either doc or ui does not exist in the changes, set both flags to false
 
-if [[ "$changed_dir" == "website" ]]; then
+if [[ $change_count -gt 2 && $change_count -ne 0 ]]; then
```

Comment:
do we need both of these conditions?

Reply:
in fact, because of the [comment above](https://github.com/hashicorp/vault/pull/21620/files#r1255999871), the second condition can never be false